openshift/README.md

```diff
@@ -0,0 +1,130 @@
+# Build and deploy on OpenShift Container Platform
```

Comment:
A link to this doc from https://github.com/strongloop/loopback4-example-shopping/blob/master/README.md would be helpful. Please create an appropriately named section (inspiration https://github.com/strongloop/loopback4-example-shopping/blob/master/README.md#deploy-to-cloud-as-microservices).

@johanschvn @mrmodise

Reply:
@hacksparrow @mrmodise what do you think about adding this after the Deploy to Cloud as Microservices section: 
```
## Build and deploy on Red Hat OpenShift

You can find [instructions](openshift/README.md), Dockerfiles and resource
definition files for building and deploying on Red Hat OpenShift Container
Platform in the openshift directory.
```

Reply:
LGTM